components/consumers/pdf/main.go

```diff
@@ -82,68 +85,79 @@ func sendToS3(filename, bucket, region string) error {
 		Body:   bytes.NewReader(data),
 	})
 	if err != nil {
-		return fmt.Errorf("unable to upload %q to %q: %w", filename, bucket, err)
+		return errors.Errorf("unable to upload %s to %s: %w", filename, bucket, err)
 	}
 
-	fmt.Printf("Successfully uploaded %q to %q\n", filename, bucket)
+	pdfFilename := strings.Replace(filename, ".html", "", -1) + ".pdf"
+	_, err = uploader.Upload(&s3manager.UploadInput{
+		Bucket: aws.String(bucket),
+		Key:    aws.String(pdfFilename),
+		Body:   bytes.NewReader(pdfBytes),
+	})
+	if err != nil {
+		return errors.Errorf("unable to upload %s to %s: %w", filename, bucket, err)
+	}
+
+	slog.Info("uploaded", "filename", filename, "pdf filename", pdfFilename, "to", "bucket", bucket, "successfully")
 	return nil
 }
 
-func buildPdf(data any) (string, error) {
+func buildPdf(data any) (string, []byte, error) {
 	tmpl := template.Must(template.ParseFiles("default.html"))
 	cleanupRun := func(msg string, cleanup func() error) {
 		if err := cleanup(); err != nil {
-			log.Printf(msg, err)
+			slog.Error(msg, "error", err)
```

Comment:
since you are fixing this, it would be a good idea to join the error with whatever other errors have been generated by the function and return it. you would need to convert the error return value into a named value for this